.github/workflows/ci_verify_pr_contains_issue.yaml

```diff
@@ -0,0 +1,16 @@
+name: VerifyIssue
+
+on:
+  pull_request:
+    types: [edited, synchronize, opened, reopened]
+  check_run:
```

Comment:
why this one?
ca the PR change via any CI action? and if so pls write it in oomment :]

Reply:
I am following the template from this Github. I will work out other features from my fork this weekend